layouts/community/list.html

```diff
@@ -60,7 +60,7 @@ <h2>Learn and Connect</h2>
             for more details on how the meeting is run, the
             <a href="https://www.youtube.com/playlist?list=PLsc4WKPCP34Q6RFVeqEVXJshpAReTmgUj">archives</a>
             for recordings of past meetings, and
-            <a href="https://docs.google.com/document/d/e/2PACX-1vSDHW7gvG8eO6aIxaIVPrZSqYYhtRDb5W1imnbpM4QRYNPsTwEO1fU5z7SEhVIFa4YqWJeSRJ9tcXYS">here</a>
+            <a href="https://docs.google.com/document/d/e/2PACX-1vSDHW7gvG8eO6aIxaIVPrZSqYYhtRDb5W1imnbpM4QRYNPsTwEO1fU5z7SEhVIFa4YqWJeSRJ9tcXYS/pub">here</a>
```

Comment:
👍 I verified this link can be seen in a private browsing tab:

<img width="986" height="841" alt="Image" src="https://github.com/user-attachments/assets/ee647091-a86f-47e9-9814-7bf4aa25af2a" />